### PR TITLE
Add PDFPreview Component

### PR DESCRIPTION
**Pull request by sscholle**
## Proposed changes

- Add PDFPreview Component
- Added unit Tests
- Added Example PDF File for storybook
- Added Mock PDF renderer for tests
- Updated Dependencies to help play nice with peers

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix
- [x] New feature
- [ ] Component of Major Release
- [ ] Project Management
- [ ] Quality of Life

## Checklist

_Put an `x` in the boxes that apply._

- [x] Linter passes (`yarn lint`)
- [x] Unit Test pass (`yarn jest`)
- [x] I have added at least two useful tests for new component (if applicable)
- [ ] I have incremented the package version and tagged the last commit of the PR with the same version number example: [0.1.29]

## Relevant Issues
- https://github.com/bonlineza/objectives-and-key-results/issues/18
- https://github.com/bonlineza/objectives-and-key-results/issues/17
- https://github.com/bonlineza/objectives-and-key-results/issues/2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,6 +81,7 @@
     "prop-types": "^15.7.2",
     "react": "~16.8.1",
     "react-dom": "~16.8.1",
+    "react-pdf-js": "^5.1.0",
```
Comment (robguy21) [inline package.json]: this should be a `peerDependancy` read more about it (here)[https://nodejs.org/en/blog/npm/peer-dependencies/#using-peer-dependencies]

File: package.json [hunk 2]
```diff
@@ -79,8 +81,6 @@
     "postcss": "^7.0.17",
     "prettier": "^1.14.3",
     "prop-types": "^15.7.2",
-    "react": "~16.8.1",
-    "react-dom": "~16.8.1",
```
Comment (robguy21) [inline package.json]: these probably still need to be in `devDependancies`

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "@bonlineza/b-lib",
   "version": "0.1.9",
   "scripts": {
-    "start": "start-storybook -p 9001 -c .storybook",
+    "start": "start-storybook -p 9001 -c .storybook -s ./public",
```
Comment (sscholle) [inline package.json]: add public folder to server for local files to be loaded in storybook

File: package.json [hunk 4]
```diff
@@ -100,11 +101,14 @@
   "jest": {
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
-      "\\.(css|less|sass|scss|postcss)$": "identity-obj-proxy"
+      "\\.(css|less|sass|scss|postcss)$": "identity-obj-proxy",
+      "../Preview/ControlItem": "<rootDir>/__mocks__/fileMock.js",
+      "react-pdf-js": "<rootDir>/__mocks__/react-pdf-js.js"
```
Comment (sscholle) [inline package.json]: mock actual pdf class in jest, we don't need to test it's functionality

**Pull request merged**